src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,43 +1,66 @@
 import React from 'react';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  todo: Todo | null;
```

Comment:
```suggestion
  todo: Todo;
```
you render this component only if todo exists